System Tests/Environment/Environment.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+struct Environment {
+    
+    // MARK: - Properties
+    
+    static let instanceLocator: String = Environment.variable(named: "INSTANCE_LOCATOR", ciValue: CI.instanceLocator)
+    
+    // MARK: - Private methods
+    
+    private static func variable(named name: String, ciValue: String) -> String {
+        guard let value = ProcessInfo.processInfo.environment[name], value != ProcessInfo.unspecifiedValue else {
+            assert(!ciValue.hasPrefix("$(") && !ciValue.hasSuffix(")"), "Please specify the value of the variable in either the enviroment variables section of your current run scheme or inject the value using Swift Variable Injector from Bitrise.")
+            return ciValue
+        }
+        
+        return value
+    }
```

Comment:
Its not straight forward to ascertain what exactly is going on here.   I _think_ the codes been update so that its now possible to define your instance locator either in the scheme or in the `CI.swift` file, if thats the case perhaps you could add a couple of additional methods to make it more obvious like...
`private static environmentVariable(fromSchemeNamed: ) -> String?`
and 
`private static environmentVariable(fromCISwiftFile: ) -> String?`

Reply:
Sure, I can do that.

Reply:
👍  ta

Reply:
Ace, this is so much better :-)